### ISSUE TITLE
[Feature]: Support 2FA vertification for login and sensitive operations

### ISSUE THREAD

**Issue by fu050409** [swpu-acm/online-judge#65]

**Comment by HsiangNianian** [comment]
I think this issue's priority must be after the mail-piece server, so you may have to draft a mail-piece server issue first :D

**Comment by HsiangNianian** [comment]
I think this issue's priority must be after the mail-piece server, so you may have to draft a mail-piece server issue first :D

**Comment by fu050409** [comment]
> I think this issue's priority must be after the mail-piece server, so you may have to draft a mail-piece server issue first :D

Yeah, you should be right about that. Even though enabling two-factor authentication doesn't require email, we don't even have any way of confirming a user's identity right now.

We don't have the enough time to spend on this at the moment, so I wonder if you have the time to get this done?
